if ilse and logical operator

```diff
@@ -0,0 +1,14 @@
+#include<stdio.h>
+int main()
+{
+int anjali,anchal,ayush;
+printf("enter the age of anjali,anchal and ayush");
+scanf("%d %d %d",&anjali, &anchal ,&ayush);
```

Comment:
give meaningful name to the variables 
Like you can have anjali_age, anchal_age ....
Currently its confusing.